components/ResultsPageComponents/CommuteDistanceDistribution/CommuteDistanceDistributionChart.jsx

```diff
@@ -26,7 +26,7 @@ export default function CommuteDistanceDistributionChart({ title, data }) {
     },
     colors: ["#044B7F"],
     credits: {
-      enabled: false
+      enabled: false,
```

Comment:
do we need a comma here?

Reply:
I think it's fine, with or without a trailing comma 👍🏽